ironfish/src/wallet/account/account.ts

```diff
@@ -42,6 +43,12 @@ export function AssertMultiSig(account: Account): asserts account is MultiSigAcc
   )
 }
 
```

Comment:
This should look like `AssertMultiSig` above, if we are just going to be using it with accounts we might as well put it on the account.

Reply:
Also we might as well rename this to `AssertSignerMultiSig`